pyproject.toml

```diff
@@ -61,7 +61,7 @@ dev = [
     "Faker>=18.0.0",
     "hypothesis>=6.0.0",
     "jupyterlab>=3.0.0",
-    "cognite-toolkit==0.5.20",
+    "cognite-toolkit==0.5.87",
```

Comment:
Upgraded Toolkit as this is what I use to deploy the data model with.

Reply:
I'll let it slide as there are no other logical changes in this PR, but we prefer dependency updates to happen in their own PRs.